### DISCUSSION TITLE
Welcome Notification Email Queueable

### DISCUSSION THREAD

**Discussion by nexgendevs** [spatie/laravel-welcome-notification#49] [Q&A]
Hi All,

I have implemented this and now when I create users from the admin interface, the email goes immediately. I want the email to be queuable. Is there any way to do that?

The answer might be very obvious, but I cannot find it anywhere.

**Comment by c4ps-lock3d** [comment]
just add `implements ShouldQueue` to WelcomeNotification class (WelcomeNotification.php)

`<?php

namespace Spatie\WelcomeNotification;

use Carbon\CarbonInterface;
use Illuminate\Foundation\Auth\User;
use Illuminate\Notifications\Messages\MailMessage;
use Illuminate\Notifications\Notification;
use Illuminate\Support\Facades\Lang;
use Illuminate\Support\Facades\URL;
use Illuminate\Contracts\Queue\ShouldQueue;
use Illuminate\Bus\Queueable;

class WelcomeNotification extends Notification implements ShouldQueue
{
    use Queueable;
    // ...`
